generated/nexus.ts

```diff
@@ -1,199 +0,0 @@
-/**
- * This file was generated by Nexus Schema
- * Do not make changes to this file directly
- */
```

Comment:
Where did this file go?

Reply:
not relevant anymore due to https://github.com/makerdao-ses/switchboard-boilerplate/pull/16#discussion_r1143081177 

initially i just yeeted the generation from new lib into the same folder in order to see if this would work and left it there because there was a topic in the linked thread i wanted to go over faster.